charts/linkerd-control-plane/values.yaml

```diff
@@ -227,6 +227,8 @@ proxyInit:
   closeWaitTimeoutSecs: 0
   # -- Allow overriding the runAsNonRoot behaviour (<https://github.com/linkerd/linkerd2/issues/7308>)
   runAsRoot: false
+  # -- This value is used only if runAsRoot is false; otherwise runAsUser will be 0
+  runAsUser: 65534
```

Comment:
Is `65534` a special user or just arbitrary in this case?

Reply:
Same ID used in the proxy-init Dockerfile